fili-core/src/main/java/com/yahoo/bard/webservice/util/Utils.java

```diff
@@ -184,4 +188,14 @@ public static void omitField(JsonNode node, String fieldName, ObjectMapper mappe
     public static <T, U, V> ImmutablePair<T, V> withRight(ImmutablePair<T, U> pair, V right) {
         return new ImmutablePair<>(pair.getLeft(), right);
     }
+
+    /**
+     * Create metrics from instance descriptors and store in the metric dictionary.
+     *
+     * @param metricDictionary  The dictionary to store metrics in
+     * @param metrics  The list of metric descriptors
+     */
+    public static void addToMetricDictionary(MetricDictionary metricDictionary, List<MetricInstance> metrics) {
```

Comment:
`addToMetricDictionary` is used in many places such as `fili-wikipedia-example` and `fili-generic-example`. This is more like an API util methods, so it's moved to here in `Utils` class